packages/fabric8-ui/src/app/space/add-app-overlay/add-app-overlay.component.spec.ts

```diff
@@ -216,15 +212,6 @@ describe('AddAppOverlayComponent', () => {
       expect(component.applications).toEqual([]);
     });
 
-    it('should retieve applications if the current space is defined', () => {
```

Comment:
Can we keep these test cases now that you've found a work around to the list of applications.